### DISCUSSION TITLE
Support Bubble doesn't show on Mobile

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [spatie/laravel-support-bubble#15] [Bugs]
The support bubble doesn't show on mobile layouts.

**Comment by freekmurze** [comment]
It does for our apps. Investigate what the problem is for your app, and if it is causee by our package, submit a PR.

**Comment by abishekrsrikaanth** [comment]
@freekmurze, I am using both brave and chrome browsers on mobile and both don't show the support bubble. I have attached the screenshots for the same.

<img width="282" alt="brave" src="https://user-images.githubusercontent.com/1639302/132260052-f2a3de9a-4177-4c59-aa14-3d6b49c539b8.png">

<img width="399" alt="chrome" src="https://user-images.githubusercontent.com/1639302/132260049-165b74db-4644-4206-a882-c15059a6f8c5.png">

**Reply by freekmurze** [reply]
I do see it at Flare. Could you try cleaning your cache? @AlexVanderbist do we show the bubble now for each authenticated user?
<img width="285" alt="CleanShot 2021-09-06 at 23 09 08" src="https://user-images.githubusercontent.com/483853/132260663-b473e259-fc12-4028-be70-630449ce5249.png">

**Reply by abishekrsrikaanth** [reply]
I tried to clear the cache. I tried on the iPad and it worked fine, but not on mobile devices. Could it be that the `block` class for the bubble is only set from `sm`

<img width="728" alt="CleanShot 2021-09-07 at 02 53 18@2x" src="https://user-images.githubusercontent.com/1639302/132261253-4271065b-7be3-4b92-b9b2-04cec28b371f.png">

Removing `hidden` does show the bubble on the mobile devices...

**Reply by AlexVanderbist** [reply]
We just don't show the bubble on mobile because it's not styled for mobile 😅 Feel free to PR mobile styles and remove the `hidden sm:block` classes

**Reply by abishekrsrikaanth** [reply]
@AlexVanderbist Thanks, it would be great to document that on the README file, It took me a while to actually figure out if there was anything else that was going wrong with the package or my code to why it wasn't working on mobile. I went through the code to understand what else was causing it. I will try to work on a PR for mobile screens.
